docs/build

```diff
@@ -9,4 +9,4 @@ test -t 1 && OPTS='-it' || OPTS=''
 SPHINX_IMAGE=${SPHINX_IMAGE:-ghcr.io/trinodb/build/sphinx:3}
 
 docker run --rm $OPTS -e TRINO_VERSION -u $(id -u):$(id -g) -v "$PWD":/docs $SPHINX_IMAGE \
-  sphinx-build -j auto -b html -W -d target/doctrees src/main/sphinx target/html
+  sphinx-build -q -j auto -b html -W -d target/doctrees src/main/sphinx target/html
```

Comment:
does this hide only progress or failures as well? (no change requested).

Reply:
```
-q
    Do not output anything on standard output, only write warnings and errors to standard error.
```

Reply:
It seems that despite this flag some progress status is still being written. I think this could be triggered by the `-j auto` option which distributes work between multiple processes. I will redirect stdout to `/dev/null`, since nothing important should be output there anyway. Stderr should contain all warnings and errors.